### ISSUE TITLE
Option to truncate long toots until expanded 

### ISSUE THREAD

**Issue by swxxii** [mastodon/mastodon-ios#730]
I would love an option to truncate long toots until clicked on to expand to see the full text.  If this could be a user configurable number of words or characters that would be ideal.

**Comment by FediVideos** [comment]
Yeah, this is already the standard on the web interface and works really well. Would be great to have this on the official apps too.

**Comment by j-f1** [comment]
Given the popularity of https://infosec.exchange/@epixoip/109585049354200263 I agree that this is necessary.
